packages/components/src/combobox-control/index.tsx

```diff
@@ -78,10 +78,12 @@ const getIndexOfMatchingSuggestion = (
  * 	{
  * 		value: 'normal',
  * 		label: 'Normal',
+ * 		disabled: true,
```

Comment:
As a new API it definitely makes sense to include a CHANGELOG entry for this PR.

Reply:
That's correct. It just needs to be moved to "Unreleased" though, that's why the check is failing.